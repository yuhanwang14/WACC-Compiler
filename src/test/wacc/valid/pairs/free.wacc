# Create and free a simple pair

# Output:

# Program:

begin
  pair(int, char) a = newpair(10, 'a') ;
  free a
end

