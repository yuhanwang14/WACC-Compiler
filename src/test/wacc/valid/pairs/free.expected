 1
 2
 3
 4
 5
 6
 7
 8
 9
10
11
12
# Create and free a simple pair

# Output:

# Program:

begin
  pair(int, char) a = newpair(10, 'a') ;
  free a
end

 